package.json

```diff
@@ -7,6 +7,8 @@
     "start": "next start"
   },
   "dependencies": {
+    "@next/swc-darwin-x64": "^12.3.1",
+    "@ramonak/react-progress-bar": "^5.0.2",
```

Comment:
delete both of these lines? it doesn't seem like you're using the react-progress-bar i think